core/src/ledger_cleanup_service.rs

```diff
@@ -32,6 +32,9 @@ pub const DEFAULT_PURGE_SLOT_INTERVAL: u64 = 512;
 // Delay between purges to cooperate with other blockstore users
 pub const DEFAULT_DELAY_BETWEEN_PURGES: Duration = Duration::from_millis(500);
 
+// Compacting at a slower interval than purging helps keep IOPS down
+const DEFAULT_COMPACTION_SLOT_INTERVAL: u64 = DEFAULT_SLOTS_PER_EPOCH;
```

Comment:
I wouldn't tie it directly to the epoch value. We can make it a number around the epoch, but I don't see how having shorter or longer epochs means you need to purge more or less.

Reply:
sgtm, I used `TICKS_PER_DAY / DEFAULT_TICKS_PER_SLOT` instead